wikitemplate-android.md

```diff
@@ -7,6 +7,7 @@
 ## Startup
 
 - [ ] Verify that Brave is only contacting `*.brave.com` endpoints on first launch using either `Charles Proxy`, `Fiddler` or `Wireshark` (or a similar application)
+  - [ ] Verify that opening a NTP doesn't trigger any outbound connections related to widgets without user interaction
```

Comment:
Not sure if it's necessary for Android. We already have https://github.com/brave/brave-browser/issues/15167 which has been the case for a while now and the only widget is Binance which is disabled by default. Doesn't hurt to check this but since we have other outbound connections that are not proxied/blocked

Reply:
I'll keep it just in case. Doesn't hurt to check to see if there's anything new that's reaching out that we might not know about. Specifically during chromium bumps. Regarding NTP, it's worth checking not just for crypto widgets, but in general to make sure that simply opening a NTP doesn't automatically create outbound connections that the user hasn't initiated.